packages/block-library/src/archives/index.php

```diff
@@ -62,7 +62,9 @@ function render_block_core_archives( $attributes ) {
 				break;
 		}
 
-		$block_content = '<label for="' . esc_attr( $dropdown_id ) . '">' . esc_html( $title ) . '</label>
+		$show_label = ( empty( $attributes['showLabel'] ) ) ? ' screen-reader-text' : '';
+
+		$block_content = '<label for="' . esc_attr( $dropdown_id ) . '" class="wp-block-archives__label' . $show_label . '">' . esc_html( $title ) . '</label>
```

Comment:
Nit: I know this was from before, but I don't see any reason for calling `esc_attr` for `$dropdown_id`.